labcodes/labstorybook
# Feature/add favicon

This PR will add favicon to project.

**How to test:**
- Run the project and access `http://localhost:8000`
- Check if the Labcodes icon is shown in the browser tab as below:
![image](https://user-images.githubusercontent.com/29437811/79374075-7caa1a80-7f2d-11ea-9cb9-f9210e7b5fab.png)
